src/commands/endpoints/index.ts

```diff
@@ -26,30 +26,38 @@ export default class EndpointsIndex extends BaseCommand {
     const database = args.database || await fetcher(this.heroku, flags.app)
     const {body: res} = await this.heroku.get<any>(`/private-link/v0/databases/${database}`, this.heroku.defaults)
 
-    cli.styledHeader(`Trusted VPC Endpoints for ${color.cyan(database)}`)
-    if (res.connections.length > 0) {
-      cli.styledObject({
-        Status: res.status,
-        'Service Name': res.service_name,
-        'Whitelisted Accounts': res.whitelisted_accounts,
-        Connections: res.connections
-      })
-    } else if (res.service_name) {
+    if (res.status === 'Provisioning') {
+      cli.log()
+      cli.log(`The Trusted VPC Endpoint is now being provisioned for ${color.cyan(database)}.`)
+      cli.log(`Run ${color.cyan('heroku endpoints:wait')} to check the creation process.`)
+    } else {
+      cli.styledHeader(`Trusted VPC Endpoint status for ${color.cyan(database)}`)
       cli.styledObject({
         Status: res.status,
-        'Service Name': res.service_name,
-        'Whitelisted Accounts': res.whitelisted_accounts,
+        'Service Name': res.service_name || 'Provisioning',
       })
-      cli.log(`
-Currently there are no active connections for your Trusted VPC Endpoint.
-
-Please follow these instructions(https://devcenter.heroku.com/articles/endpoints) to fully setup your endpoint.
-      `)
-    } else {
-      cli.log(`
-A Trusted VPC Endpoint has not been created for this database.
 
-In order to create one, run $ heroku endpoints:create`)
+      if (res && res.whitelisted_accounts.length > 0) {
+        cli.log()
+        cli.styledHeader('Whitelisted Accounts')
+        cli.table(res.whitelisted_accounts, {
+          arn: {header: 'ARN'},
+          status: {}
+        })
+      }
+
+      if (res && res.connections.length > 0) {
+        cli.log()
+        cli.styledHeader('Connections')
+        cli.table(res.connections, {
+          endpoint_id: {header: 'Endpoint ID'},
+          owner_arn: {header: 'Owner ARN'},
+          status: {}
+        })
+      } else if (res.status === 'Operational' && res.connections.length === 0) {
+        cli.log('Your Trusted VPC Endpoint is now operational.')
+        cli.log(`You must now copy the ${color.cyan('Service Name')} and follow the rest of the steps listed in https://devcenter.heroku.com/articles/setting-up-a-trusted-vpc-endpoint?preview=1.`)
```

Comment:
This has moved: https://devcenter.heroku.com/articles/trusted-vpc-endpoints

Also, I don't think we should include the `preview=1` url param. It should be published soon.